lstchain/io/io.py

```diff
@@ -815,7 +819,7 @@ def recursive_copy_node(src_file, dir_file, path):
             recursive_path = os.path.join(recursive_path, p)
 
 
-def write_calibration_data(writer, mon_index, mon_event, new_ped=False, new_ff=False):
+def write_calibration_data(writer, table_group, mon_index, mon_event, new_ped=False, new_ff=False):
```

Comment:
Why this new argument when you then always pass `telescope/monitoring`?

Reply:
Honestly, I do not remember, I can indeed take it out